libs/surfaces/console1/console1.cc

```diff
@@ -187,16 +185,30 @@ Console1::begin_using_device ()
 	Glib::RefPtr<Glib::TimeoutSource> periodic_timer = Glib::TimeoutSource::create (100);
 	periodic_connection = periodic_timer->connect (sigc::mem_fun (*this, &Console1::periodic));
 	periodic_timer->attach (main_loop ()->get_context ());
+	connect_session_signals ();
+	connect_internal_signals ();
+	create_strip_inventory ();
 
 	DEBUG_TRACE (DEBUG::Console1, "************** begin_using_device() ********************\n");
 
-	create_strip_inventory ();
-	connect_internal_signals ();
-	stripable_selection_changed ();
-
 	return MIDISurface::begin_using_device ();
 }
 
+int
+Console1::stop_using_device ()
+{
+	DEBUG_TRACE (DEBUG::Console1, "stop_using_device()\n");
+	if (!_in_use)
+		return 0;
+	blink_connection.disconnect ();
+	periodic_connection.disconnect ();
+	stripable_connections.drop_connections ();
+	session_connections.drop_connections ();
```

Comment:
This line can go away
Session connections are unset by `MIDISurface::stop_using_device ();`